frontend/src/routes/FacilityBooking/ViewFacility/index.tsx

```diff
@@ -31,6 +31,7 @@ import PullToRefresh from 'pull-to-refresh-react'
 import dayjs from 'dayjs'
 import Calendar from 'react-calendar'
 import './calendar.css'
+import BookingCard from '../../../components/BookingCard'
```

Comment:
is this used in this page? might not be necessary now since we will redo the page

Reply:
Resolved in commit 6c4688c5a5c1853aa8ff09a0d49fdb749f87bdbb